tools/run_tests/run_interop_tests.py

```diff
@@ -1029,5 +1096,8 @@ def aggregate_http2_results(stdout):
   dockerjob.finish_jobs([j for j in server_jobs.itervalues()])
 
   for image in docker_images.itervalues():
-    print('Removing docker image %s' % image)
-    dockerjob.remove_image(image)
+    if not args.manual_run:
+      print('Removing docker image %s' % image)
+      dockerjob.remove_image(image)
+    else:
+      print('Preserving docker image: %s' % image)
```

Comment:
Unless I missed anything, the "docker build" will happen every time when you run.  If nothing has changed, it should be an noop. 

Having a --save_docker_image flag is fine but for version compatibility, but it is important that we provide clear signal what images are built from a run without requiring any guess work.